src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskForStreamingIT.java

```diff
@@ -303,4 +307,135 @@ public void testSinkTaskWithMultipleOpenClose() throws Exception {
 
     assert partitionsInTable.size() == 2;
   }
+
+  @Test
+  public void testTopicToTableRegex() {
+    Map<String, String> config = TestUtils.getConfForStreaming();
+
+    testTopicToTableRegexMain(config);
+  }
+
+  // runner for topic to table regex testing, used to test both streaming and snowpipe scenarios.
```

Comment:
This is a little hacky, but I'd rather do this than copy-paste the test twice. I tried moving it to testUtils, however the scope of SnowflakeSinkConnectorConfig.TOPICS_TABLES_MAP and a couple other variables were not accessible.

Any thoughts on how to improve this?